RestaurantReservation/Services/CustomerService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using RestaurantReservation.Db.Models;
+using RestaurantReservation.Db;
+using Microsoft.EntityFrameworkCore;
+namespace RestaurantReservation.Services
+{
+    public class CustomerService
+    {
+        private readonly RestaurantReservationDbContext _context;
+
+        public CustomerService(RestaurantReservationDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task CreateCustomerAsync(Customer customer)
+        {
+            if (customer == null)
```

Comment:
Maybe we should check if the customer is null out, I mean Why there is a need to call it if the entity is null?